client/src/components/existing-request/existing-request-display.vue

```diff
@@ -32,8 +32,8 @@
             <v-col cols="9" class="py-0">
               <v-row dense>
                 <v-col cols="12">
-                  <span>Last Update:</span>
-                  &nbsp;{{ lastUpdate }}
```

Comment:
Please delete this getter if it's no longer used.

Reply:
Getter deleted